contrib/services/systemd/dcrd.service

```diff
@@ -27,6 +27,7 @@ Restart=on-abnormal
 RestartSec=10s
 
 # strict privilege
+InaccessiblePaths=/usr/local/sbin /usr/local/bin /usr/sbin /usr/bin /sbin /bin
```

Comment:
restricting read access to /usr and/or /usr/local, where Go is typically installed from packages, may result in missing information in stack traces

Reply:
nvm, all of this should be baked into the executable.